### DISCUSSION TITLE
Specify timeframe (compare to previous period)

### DISCUSSION THREAD

**Discussion by RayLation** [kalkih/mini-graph-card#706] [Q&A]
Hello,

Based on my reading, I don't think this is possible, but wish to confirm.

If I only want to see data for a specific period. Example of usage: Have 2 lines in the graph: Last month vs Current Month.

Is it possible to do it ? Or filter the data in any other way before using the graph?

Thanks!

**Comment by jlsjonas** [comment]
A feature to allow comparing to a previous period (if data is available outside of statistics) is scheduled _soon_, it's something I personally also want to do with several data items.

Stay tuned!

**Reply by settercoly** [reply]
Can this be done now?

**Reply by ildar170975** [reply]
no

**Reply by settercoly** [reply]
Ok

**Reply by ApiFelix** [reply]
and is it possible to choose to display a selected period ? current day ...

thanks

**Reply by ildar170975** [reply]
> a selected period

No, already answered.

> current day

There was an example with using config-template-card, try find it in Issues.
